sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -172,7 +180,9 @@ def __init__(self, fully_qualified_namespace, credential, **kwargs):
             super(ServiceBusSender, self).__init__(
                 fully_qualified_namespace=fully_qualified_namespace,
                 credential=credential,
-                entity_name=entity_name,
+                entity_name=str(entity_name),
```

Comment:
why here we changed to cast it to a str?

Reply:
oh, I see that it's what we're doing in async, yea, I think cast to str does no harm, we can do it